include/open62541/common.h

```diff
@@ -175,8 +175,10 @@ typedef enum {
 
 typedef enum {
     UA_SECURECHANNELSTATE_FRESH = 0,
+    UA_SECURECHANNELSTATE_REVERSE_LISTENING,
     UA_SECURECHANNELSTATE_CONNECTING,
     UA_SECURECHANNELSTATE_CONNECTED,
+    UA_SECURECHANNELSTATE_REVERSE_CONNECTED,
```

Comment:
The new states work great!
Please adjust UA_SecureChannel_isConnected so that a reverse-listening SecureChannel is also considered "connected".
(So socket->shutdown is called on client->channel.connectionId.)

```c
UA_Boolean
UA_SecureChannel_isConnected(UA_SecureChannel *channel) {
    return (channel->state >= UA_SECURECHANNELSTATE_CONNECTING &&
            channel->state < UA_SECURECHANNELSTATE_CLOSING);
}
```

Reply:
Done